### PR TITLE
[Fix]fix check_env

### PR DESCRIPTION
**Pull request by VVsssssk**
## Motivation

Fix check_env.py script to check codebase version.

## Modification

Fix check_env.py

## Use cases (Optional)

python tools/check_env.py

### CODE REVIEW

File: tools/check_env.py [hunk 1]
```diff
@@ -54,8 +55,81 @@ def check_backend():
...
+
+
+def check_codebase():
+    ops = {
+        '>=': (lambda x, y: x >= y),
+        '==': (lambda x, y: x == y),
+        '<=': (lambda x, y: x <= y),
+        '': (lambda x, y: True)
+    }
+    with open('../requirements/optional.txt', 'r') as file:
```
Comment (RunningLeon) [inline tools/check_env.py:73]: file path may not exist if calling on different directory.

File: tools/check_env.py [hunk 2]
```diff
@@ -54,8 +55,81 @@ def check_backend():
     logger.info(f'openvino_is_avaliable: {openvino_apis.is_available()}')
 
 
+def get_version(s):
+    return ''.join([c for c in s if c.isdigit() or c == '.'])
+
+
+def get_op(s):
+    return ''.join([c for c in s if c in '><='])
```
Comment (grimoire) [inline tools/check_env.py:63]: Use Regular Expression instead of parse string.

File: tools/check_env.py [hunk 3]
```diff
@@ -54,8 +55,81 @@ def check_backend():
...
+        '': (lambda x, y: True)
+    }
+    with open('../requirements/optional.txt', 'r') as file:
+        try:
+            import mmcls
+        except ImportError:
+            mmcls_version = None
+        else:
+            mmcls_version = mmcls.__version__
+        requirement = file.readline()
```
Comment (grimoire) [inline tools/check_env.py:80]: Not safe to check packages one by one.
The package order in `optional.txt` might change in the future.

Comment (RunningLeon) [reply tools/check_env.py:80]: Version requirements may change according to [PEP ](https://www.python.org/dev/peps/pep-0440/#version-specifiers) and this check may fail at some cases.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by grimoire — APPROVED** [review]

**Pull request merged**